scripts/misc/commit_checker.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+#
+# Copyright (C) 2020 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+#
+
+# This is script for verifying commit message by SDK requirements.
+# Requirements can be find in scripts/misc/commit_message_recom.txt
+
+# Saving whole commit message into file for reading below
+echo "`git log --pretty=format:'%B' -2 | sed '1d' | sed '1d' `" >> commit.log
+
+# Counting number of lines in file
+num_lines=`wc -l commit.log| cut -d'c' -f1` 
+
+# Loop for every line in file
+for (( line=1; line<=${num_lines}; line++ ))
+do
+    current_line_len=`cat commit.log| sed -n ${line}p |wc -m | sed 's/\ \ \ \ \ \ /''/g' `
+    if [ $line -eq 1 ] && [ ${current_line_len} -gt 50 ] ; then
```

Comment:
50 is too small. Consider this sentence:
`Specify the OLP SDK version in the CMake file`
it's 45 chars long. Adding a few more words would fail the check.